mappings/net/minecraft/server/network/ClientServerPlayerEntity.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_980 net/minecraft/server/network/ClientServerPlayerEntity
```

Comment:
ClientServer..?

Reply:
@Blayyke 
You mapped this as ServerPlayerEntity, however it has `@Environment(EnvType.CLIENT)` and there was a similar one for EnvType.SERVER

so I made the server-side one ServerPlayerEntity and made the client one ClientServerPlayerEntity

Reply:
In the spirit of valo names, we must keep this

Reply:
fair enough good enough for now